packages/edit-site/src/components/sidebar-dataviews/dataview-item.js

```diff
@@ -51,6 +52,7 @@ export default function DataViewItem( {
 			<SidebarNavigationItem
 				icon={ iconToUse }
 				{ ...linkInfo }
+				suffix={ <span>{ count }</span> }
```

Comment:
Is the span needed? 🤔

Reply:
The `span` is not needed, but if `count` is `0` it might mess up with the boolean logic:

https://github.com/WordPress/gutenberg/blob/e1aaee2eb146467c0b20f469c2b04f2adc7295ac/packages/edit-site/src/components/sidebar-navigation-item/index.js#L56

Passing an object (which the React element `<span />` essentially is) will always evaluate to `true`, bypassing that problem.

But again, the `span` is not needed. The same can be achieved with a `React.Fragment`:

```suggestion
				suffix={ <>{ count }</> }
```

Alternatively, we'd alter `suffix` to deal better with `0` values, but this sounds like overkill.